### DISCUSSION TITLE
Keycloak admin console interface is out of screen - Add scroll bar

### DISCUSSION THREAD

**Discussion by i62pecar** [keycloak/keycloak#32737] [Ideas]
Hello, I'm currently using Keycloak 25.0.1 and I have a problem with the user interface.

![image](https://github.com/user-attachments/assets/f7648dff-b6d2-4058-8757-f0a963cab190)

As you can see the name of the resources is too long, so the options for the resources are missing.
To be able to see the options I need to zoom out a lot, so the text is small and barely readable.

![image](https://github.com/user-attachments/assets/a929909c-59d5-4d93-a70a-3b019cd894d3)

Would it be possible to add a scroll bar in this screen so we can scroll left and right without zooming the content?

Or maybe if the resources are too long show them like "Reso...urce".

Just throwing some ideas of how this can be fixed. Thanks for your time! :)

**Comment by ssilvert** [comment]
@i62pecar Please open an issue for this.

**Reply by jonkoops** [reply]
Closed the discussion as an issue has been logged.
